Sources/Authenticator/States/ContinueSignInWithMFASetupSelectionState.swift

```diff
@@ -0,0 +1,58 @@
+//
```

Comment:
This new state should be added to the [`PreviewFactory`](https://github.com/aws-amplify/amplify-ui-swift-authenticator/blob/main/Sources/Authenticator/Utilities/PreviewFactory.swift)